ssl/ssl_ciph.c

```diff
@@ -223,7 +229,7 @@ static const SSL_CIPHER cipher_aliases[] = {
     {0, SSL_TXT_IDEA, NULL, 0, 0, 0, SSL_IDEA},
     {0, SSL_TXT_SEED, NULL, 0, 0, 0, SSL_SEED},
     {0, SSL_TXT_eNULL, NULL, 0, 0, 0, SSL_eNULL},
-    {0, SSL_TXT_GOST, NULL, 0, 0, 0, SSL_eGOST2814789CNT | SSL_eGOST2814789CNT12},
+    {0, SSL_TXT_GOST, NULL, 0, 0, 0, SSL_eGOST2814789CNT | SSL_eGOST2814789CNT12 | SSL_MAGMA | SSL_KUZNYECHIK},
```

Comment:
I think it’s not correct to use the alias SSL_TXT_GOST for Magma and Kuznyechik, which is defined as "GOST89" (/master/include/openssl/ssl.h :134). Perhaps the best solution would be to identify new alias. What do you think?

Reply:
Maybe. Maybe not.

Reply:
what do you mean with that?

Reply:
BTW: this line is way too long.

Reply:
Fixed.